<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayMediaLitItem</class>
 <widget class="QWidget" name="PlayMediaLitItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>34</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>34</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PlayMediaLitItem</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel,QPushButton{
font-family:&quot;微软雅黑&quot;;
font-size:12px;
background:transparent;
}
#label_mediaName{
color:#999999;
}
#label_playTime{
color:#DCDCDC;
}
#wgt_control{
background:transparent;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>11</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label_selected">
     <property name="minimumSize">
      <size>
       <width>4</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>4</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_mediaName">
     <property name="text">
      <string>探球网.mp4</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>226</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_playTime">
     <property name="text">
      <string>00:00:00</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wgt_control" native="true">
     <property name="styleSheet">
      <string notr="true">#button_up{
border-image: url(:/TQLive/images/live tool_icon_up.png);
}
#button_down{
border-image: url(:/TQLive/images/live tool_icon_down.png);
}
#button_del{
border-image: url(:/TQLive/images/live tool_icon_delete.png);
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="button_up">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>6</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>10</width>
          <height>6</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_down">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>6</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>10</width>
          <height>6</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_del">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>12</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>10</width>
          <height>12</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TQLive.qrc"/>
 </resources>
 <connections/>
</ui>
